packages/next/src/lib/typescript/writeAppTypeDeclarations.ts

```diff
@@ -61,8 +61,10 @@ export async function writeAppTypeDeclarations({
     distDir.replaceAll(path.win32.sep, path.posix.sep),
     'types/routes.d.ts'
   )
+  // const validatorPath = routeTypesPath.replace(/routes\.d\.ts$/, 'validator.ts')
 
   directives.push(`/// <reference path="./${routeTypesPath}" />`)
+  // directives.push(`/// <reference path="./${validatorPath}" />`)
```

Comment:
Why is this commented out?

Reply:
It's commented out because it doesn't actually need to be there -- `.next/types` is included in the tsconfig already, so including it as a triple-slash directive would just improve auto-import afaik (which is not necessary in this case).

Tbh, I should probably just remove it. My original goal was to get the TS LSP to continually validate the file, but seems like it's not possible.

Reply:
Update: removed entirely. Not necessary for type validation